src/extensions/renderer/base/coord-ele-math/coords.js

```diff
@@ -333,7 +333,7 @@ BRp.getAllInBox = function( x1, y1, x2, y2 ){
         includeLabels: false
       } );
 
-      if( math.boundingBoxesIntersect( boxBb, nodeBb ) ){
+      if( math.boundingBoxesIntersect( boxBb, nodeBb ) && !math.boundingBoxInBoundingBox( nodeBb, boxBb ) ){
```

Comment:
@maxkfranz In testing I have not noticed any weirdness with normal nodes. I have no idea how it is set up but `!math.boundingBoxInBoundingBox( nodeBb, boxBb )` is true if `boxBb` inside `nodeBb`. If `nodeBb` is essentially a point value, this condition is virtually impossible to meet.